draft-ietf-oauth-transaction-tokens.md

```diff
@@ -403,7 +403,7 @@ JWT claims as well as defines new claims. These claims are described below:
 : REQUIRED A unique transaction identifier as defined in Section 2.2 of {{RFC8417}}.
 
 `sub`:
-: REQUIRED A unique identifier for the subject within the context of the `aud` Trust Domain.
+: REQUIRED This claim represents the principal of the transaction as defined by Section 4.1.2 of {{RFC7519}}. The value MUST be unique within the context of the `aud` Trust Domain. Unlike OpenID Connect, the `sub` claim is NOT associated with the `iss` claim.
```

Comment:
> Unlike OpenID Connect, the `sub` claim is NOT associated with the `iss` claim.

We got comments in WGLC about the appropriateness fo describing wht something is not. Is there a specific reason to call this out here (I have a recollection of removing a similar sentence earlier)?

Reply:
@gffletch - yeah, I did not mean to remove the REQUIRED section. Perhaps better for you to just make the change in the PR rather than chancing it with the committing the suggestion.